lms/views/lti/deep_linking.py

```diff
@@ -49,20 +62,96 @@
     schema=DeepLinkingLTILaunchSchema,
 )
 def deep_linking_launch(context, request):
-    request.find_service(name="application_instance").get_current().update_lms_data(
-        request.params
-    )
+    """Handle deep linking launches."""
+    application_instance = request.find_service(
+        name="application_instance"
+    ).get_current()
+    application_instance.update_lms_data(request.params)
 
     context.get_or_create_course()
 
     request.find_service(name="lti_h").sync([context.h_group], request.params)
 
     context.js_config.enable_file_picker_mode(
-        form_action=request.params["content_item_return_url"],
+        form_action=request.parsed_params["content_item_return_url"],
         form_fields={
             "lti_message_type": "ContentItemSelection",
-            "lti_version": request.params["lti_version"],
+            "lti_version": request.parsed_params["lti_version"],
         },
     )
 
+    if application_instance.lti_version == "1.3.0":
```

Comment:
This conditional will disappear in the next PR where we enable the same flow for v11.